src/extra_model_paths.yaml

```diff
@@ -0,0 +1,11 @@
+comfyui:
+     base_path: /runpod-volume
```

Comment:
This file is a special file for ComfyUI that allows a user to configure custom paths. Here we're pointing to the default mount point that Runpod gives to Serverless pods.